src/query/list_file.c

```diff
@@ -5527,7 +5537,6 @@ qfile_clear_uncommited_list_cache_entry (THREAD_ENTRY * thread_p, int tran_index
   lent = qfile_List_cache.tran_list[tran_index];
   while (lent)
     {
-      lent->uncommitted_marker = false;
```

Comment:
Is it okay if uncommitted_marker is not initialized to false?

Reply:
the following functions are not used:
  qfile_add_uncommitted_list_cache_entry
  qfile_clear_uncommited_list_cache_entry
  qfile_delete_uncommited_list_cache_entry
for transaction contention problem between cached result.